components/general/Filters.tsx

```diff
@@ -20,9 +22,10 @@ const Filters = ({
 
   return (
     <ul className={stls.filters}>
-      <SearchField/>
       <li>
-        <h4 className={stls.title}>Тип программы</h4>
+          <SearchField/>
+        <h4 className={stls.title}>Формат обучения</h4>
```

Comment:
I know the task was to replace the typo, but there is an update now that suggest we do this
```suggestion
        <h4 className={stls.title}>Тип программы</h4>
```